src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -298,6 +308,7 @@ export default function TransactionFeedV2() {
   const allowedNetworkForTransfers = useAllowedNetworksForTransfers()
   const address = useSelector(walletAddressSelector)
   const localCurrencyCode = useSelector(getLocalCurrencyCode)
+  const isRefreshingBalances = useSelector((state) => state.home.loading)
```

Comment:
@kathaypacific should this be moved to a named selector? Or is it fine to use explicit selectors from time to time if it's simple enough? Asking for myself for the future :)